modules/cram/templates/whatshap.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -euo pipefail
+
+create_ped () {
+  echo -e "!{pedigreeContent}" > "!{pedigree}"
+}
+
+filter_bams () {
+  # filter all reads without readgroups
+  count=$(${CMD_BCFTOOLS} query -l "!{vcf}" | wc -l)
+  if [ "${count}" -gt 1 ]; then
```

Comment:
```suggestion
  if [ "${count}" -gt 0 ]; then
```

Reply:
no, in case of a single patient all reads in the cram are assumed to be of that patient